### PR TITLE
feat(getDbInstance): Added getDbInstance method

### PR DESCRIPTION
**Pull request by BorntraegerMarc**
Signed-off-by: BorntraegerMarc <marc.borntraeger@gmail.com>

### CODE REVIEW

**Comment by BorntraegerMarc** [comment]
@wzrdtales Created new PR to comply to all project requirements. Tests look OK to you?

**Comment by wzrdtales** [comment]
Looks good so far, I had a look over this again now, did you tested this for functionality?

https://github.com/BorntraegerMarc/mongodb/blob/cfe208c5805ee244560b9343490b9e5e31eca323/index.js#L306

As you call this callback here, the river would get closed before it gets to the user, this would need to keep open, which also needs some information for the user that they need to close the connection when they request an instance in that way.

**Comment by BorntraegerMarc** [comment]
@wzrdtales good catch! I've added an `if` statement there. Looks good now?

**Comment by wzrdtales** [comment]
I would rather call `prCB` directly instead of introducing a string heavy comparison which was already evaluated once before.

File: index.js [hunk 1]
```diff
@@ -345,6 +357,9 @@ var MongodbDriver = Base.extend({
           case 'updateMany':
             db.collection(collection)[command](options.query, options.update, options.options, callbackFunction);
             break;
+          case 'getDbInstance':
+            prCB(null, db); // When the user wants to get the DB instance we need to retrun the promise callback, so the DB connection is not automatically closed
```
Comment (BorntraegerMarc) [inline index.js]: @wzrdtales like this you mean?

Comment (wzrdtales) [reply index.js]: Yes, but I typo slipped in `retrun` instead of `return` :)

Comment (BorntraegerMarc) [reply index.js]: @wzrdtales Good catch 😄 Corrected it

**Comment by wzrdtales** [comment]
Looks good, merging at this point.

**Comment by BorntraegerMarc** [comment]
Thanks @wzrdtales ! Could you release a new npm version so I can test out the functionality? 😄

**Comment by wzrdtales** [comment]
later this evening yes (I'm gmt+1 though)

**Comment by wzrdtales** [comment]
Was just published as 
+ db-migrate-mongodb@1.4.0

**Pull request merged**